### PR TITLE
Add .cci.jenkinsfile

### PR DESCRIPTION
**Pull request by cgwalters**
See https://github.com/jlebon/coreos-ci

This is just a start to test, cut down from the rpm-ostree version.

### CODE REVIEW

**Comment by cgwalters** [comment]
OK...debating a bit whether to merge this as is or to try to just push forward and drop PAPR testing entirely.

**Comment by cgwalters** [comment]
Leaning a bit towards merging as is, so if someone could drop a `/lgtm` that'd be great!

File: .cci.jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// See https://github.com/jlebon/coreos-ci
+// This was originally copied from the rpm-ostree one
+@Library('github.com/coreos/coreos-ci-lib@master') _
+
+COSA_IMAGE = 'quay.io/coreos-assembler/coreos-assembler:latest'
+COSA_BUILDROOT_IMAGE = 'registry.svc.ci.openshift.org/coreos/cosa-buildroot:latest'
+
+stage("Build") {
+parallel rpms: {
```
Comment (jlebon) [inline .cci.jenkinsfile]: Maybe just `build`?

Comment (cgwalters) [reply .cci.jenkinsfile]: Heh I saw that and was going to fix it but didn't want to wait for a whole other cycle of CI runs...

**Pull request merged**